src/test/test-fs-util.c

```diff
@@ -864,14 +864,16 @@ static void test_path_is_encrypted_one(const char *p, int expect) {
 }
 
 static void test_path_is_encrypted(void) {
-        log_info("/* %s */", __func__);
+        int booted = sd_booted();
+
+        log_info("/* %s (sd_booted=%d)*/", __func__, booted);
 
         test_path_is_encrypted_one("/home", -1);
         test_path_is_encrypted_one("/var", -1);
         test_path_is_encrypted_one("/", -1);
-        test_path_is_encrypted_one("/proc", false);
-        test_path_is_encrypted_one("/sys", false);
-        test_path_is_encrypted_one("/dev", false);
+        test_path_is_encrypted_one("/proc", booted > 0 ? false : -1);
+        test_path_is_encrypted_one("/sys", booted > 0 ? false : -1);
+        test_path_is_encrypted_one("/dev", booted > 0 ? false : -1);
```

Comment:
a simpler fix might be to simply delete this whole line actually, given that /dev can be anything. Testing /sys and /proc should be enough

Reply:
I kept the test, because it's good to look at the corner cases.